tasks/collect-cosign-params/collect-cosign-params.yaml

```diff
@@ -22,7 +22,8 @@ spec:
     - name: cosign-secret-name
       type: string
       description: >-
-        Name of secret which contains AWS_DEFAULT_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and SIGN_KEY
+        Name of secret which contains AWS_DEFAULT_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY,
```

Comment:
The README for this task should probably be updated for the same change

Reply:
README doesn't mention about result of the task, should it?

Reply:
I noticed this too and decided it was ok since the README doesn't mention the variables. But yeah, we usually have the description in both places, so it probably makes sense to do it here as well.

Reply:
I checked README of other tasks, they don't contain description of `results` either. They do have description of `params` in both places.

Reply:
What I was saying is that in most cases both places have the exact same description. Results are sometimes mentioned and sometimes not. I think it's nice to have the results mentioned. Anyway, I'd say it's up to you from my POV. But since Johnny opened this thread, let's see what he says.